src/main/java/core/basesyntax/figure/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+public class Rectangle extends AbstractFigure implements Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
```

Comment:
Consider removing expressions from the constructor parameters for better readability. Instead, assign the values to local variables and pass those variables to the constructor.